src/sidebar/templates/annotation-share-dialog.html

```diff
@@ -36,7 +36,7 @@
       <i class="h-icon-clipboard btn-icon"></i>
     </button>
   </div>
-  <div class="annotation-share-dialog-msg" ng-if="vm.group && !vm.group.public && !vm.isPrivate">
+  <div class="annotation-share-dialog-msg" ng-if="vm.group && vm.group.type === 'private' && !vm.isPrivate">
```

Comment:
This logic is a bit gross but I decided that cleaning it up fell outside the scope of this PR.